tools/sidekick/config.js

```diff
@@ -101,13 +109,35 @@ function hasSchema(host) {
           action: () => {
             window.open(
               `https://search.google.com/test/rich-results?url=${encodeURIComponent(
-                window.location.href,
+                window.location.href
               )}`,
-              'check-schema',
+              'check-schema'
             );
           },
         },
       },
+      {
+        id: 'ost',
+        condition: (s) => s.isEditor(),
+        button: {
+          text: 'OST',
```

Comment:
what is the difference between this and the next one `'OST (iframe)'`?

Reply:
I was planning to show both full page and iframe approach of ost, but since real estate is too big for an iframe we agreed to keep it full page only. I have removed the dead code.